docs/api/genomicRdd.rst

```diff
@@ -152,3 +152,100 @@ Similar to ``transform``/``transformDataset``, there exists a
+than 20x when repeatedly querying a datset with genomic range filters.   
+On a high coverage alignment dataset, this partitioning strategy improved
+latency from 1-2 minutes to 1-3 seconds when looking up genomics ranges.
+
+**Saving partitioned parquet files to disk**  
+
+A ``GenomicRDD`` can be written to disk as a paritioned Parquet dataset with the
+``GenomicRDD`` function ``saveAsParitionedParquet``. The optional
+``partitionSize`` parameter  defines the width in base pairs of the partitions
+within each contig.
+
+.. code:: scala
+ data.saveAsPartitionedParquet("dataset1.adam", partitionSize = 2000000)
+
+A partitioned dataset can also be created from an input Parquet or SAM/BAM/CRAM
+file using the ADAM ``transformAlignments`` CLI, or Parquet/VCF files using the
+ADAM ``transformGenotypes`` CLI.
+
+**Loading partitioned parquet files**
+
+A GenomicRDD can be loaded from a partitioned {arquet dataset using the
+ADAMContext functions ``loadPartitionedParquetDataset`` or by using the
+``--isBinned`` options in the CLI.
+
+**Layout of Partitioned Parquet directory**
+
+An ADAM partitioned Parquet dataset is written as a three level directory hierarchy.
+The outermost directory is the name of the dataset and contains metadata files as is
+found in unparitioned ADAM Parquet datasets. Within the outer dataset directory,
+subdirectories are created with names based on the each of the genomic contigs found
+in the dataset, for example a subdir will be named ``contig=22`` for chromosome 22.
+Within each contig directory, there are subdirectories named using a computed value
+``positionBin``, for example a subdirectroy named ``positionBin=22``. Records from the
+dataset are written into Parquet files within each postionBin directory, computed
+based on startPos of the record using the calculation ``floor(start / partitionSize)``.
+For example, when using the default ``partitionSize`` of 1,000,000 base pairs, an 
+alignment record with start position 20,100,000 on chromoome 22 would be found in a 
+parquet file at the path ``mydataset.adam/contig=22/positionBin=20``. The splitting 
+of data into one or more Parquet fields in these leaf directories is automatic based on 
+parquet block size settings.
+
+.. code:: 
+
+  mySavedAdamDataset.adam
+  |
+  |-- _partitionedByStartPos
+  L-- contig=1  
+      L-- positionBin=0
+          |-- part-r-00001.parquet
+          +-- part-r-00002.parquet
+      L-- positionBin=1
+          |-- part-r-00003.parquet
+          |-- part-r-00004.parquet 
+      L-- positionBin= ( N bins ...)              
+  L--  contig= ( N contigs ... )
+      |-- (N bins ... )
+  
+  
+The existence of the file ``_partitionedByStartPos`` can be tested with the public
+function ``ADAMContext.isPartitioned(path)`` and can be used to determine explicitly
+if an ADAM Parquet dataset is partitioned using this scheme. The parition size used
+when the dataset was written to disk is stored in ``_parititionedByStartPos`` and is
```

Comment:
parititionedByStartPos &rarr; partitionedByStartPos

Reply:
fixed -thanks, I honestly need to increase my font size so I can see typos.